app/javascript/packs/pages/provider_signup/SignupStepper.jsx

```diff
@@ -137,7 +137,17 @@ export default function SignupStepper({ steps }) {
   const CurrentStep = steps[activeStep] && steps[activeStep].component;
 
   if (redirectId) {
-    return <Redirect to={`/providers/${redirectId}?success`} />
+    return (
+      <Redirect
+        push
+        to={{
+          pathname: `/providers/${redirectId}`,
```

Comment:
what are your thoughts on making a route like `/provider-signup/success` and passing the provider ID into that? My thinking is that using this route for confirmation and provider viewing could make traffic data harder to understand if we were to add something like google analytics

Reply:
Oooh. Good points.

I recall the initial concern about passing the query param `?success` to be about providers copying that URL and sharing it, thus resulting in page where users could not view the "Offer Help" button.

Does the `/provider-signup/success` route put us in the same position?

Reply:
Ah, yes. I suppose it does. I'm over-thinking, this work is great!

Tangentially related, it seems like we should add UI on the success that says "Share this link with your friends".  I'll chat with folks.

Reply:
We have a spec for it, but do not believe we have discussed URL shortening?